Source/WebInspectorUI/UserInterface/Views/QuickConsole.css

```diff
@@ -51,7 +52,8 @@
 
     align-items: flex-start;
 
-    max-height: 33vh;
+    max-height: 100%;
+    will-change: z-index; /* Workaround for <rdar://125601926> - Content renders on top of the scroll bar */
```

Comment:
This is a workaround for the same issue as #28050 where the scrollbar of the scroll container is obscured by the content within.